.vscode/settings.json

```diff
@@ -1,5 +1,17 @@
 {
   "editor.defaultFormatter": "biomejs.biome",
+  "[javascript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[json]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[jsonc]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
```

Comment:
will merge this part in separately if we dont like the idea of claimCount

This was i couldnt figure out why my format on save was defaulting to prettier over biome. Turns out i had specific file type settings like this in my user settings and because they were more specific then just editor.defaultFormatter it was using them over the workspace settings